utils/evm_loader.py

```diff
@@ -976,3 +979,15 @@ def delete_holder(self, del_key: Pubkey, acc: Keypair, signer: Keypair):
         trx = Transaction()
         trx.add(make_DeleteHolderAccount(acc.pubkey(), del_key, self.loader_id))
         return self.send_tx(trx, signer)
+
+    def drain_sol(self, from_: Keypair, to: Pubkey):
```

Comment:
let's  move it to solana_client as  you don't use here nothing connected to evm loader and neon itself